resources/sts/4.11/managed/ROSAInstallerCorePolicy.json

```diff
@@ -2,6 +2,7 @@
   "Version": "2012-10-17",
   "Statement": [
     {
+      "Sid": "DescribeResources",
```

Comment:
`"elasticloadbalancing:CreateTargetGroup",` and `"tag:UntagResources",` is in this group, should those be moved?